python/tskit/trees.py

```diff
@@ -5522,6 +5522,32 @@ def genetic_relatedness(
         :ref:`polarised <sec_stats_polarisation>`,
         and :ref:`return value <sec_stats_output_format>`.
 
+        What is computed depends on ``mode``:
```

Comment:
Below it's written sort of as if each sample set has only one chromosome (e.g., "between samples `a` and `b`") but also acknowledging that this is not true (e.g., "number of *pairwise* allelic matches"). (it's tricky to get this right!)

Reply:
Ah right, I hadn't thought about the case where each sample set can have more than one sample...
If the x[j] can take values greater than 1, we don't have the identity 2(a−c)(b−d) = [a=b]−[b=c]−[a=d]+[c=d] (used in the overleaf document). So, do we necessarily have the same interpretation?

Reply:
ah, I get it now! The number of pairwise matches is saying that we sum over the combinations of samples in different sample sets.

Reply:
Yeah - it's kinda wierd, but it's apparently the right thing for the covariance matrix. (hm: given this is the more interesting use of it, we should probably add the "covariance matrix of a quantitative trait" interpretation to the documentation?)

Reply:
Yep, I agree it would be worth emphasising this. And maybe highlighting that to get the covariance between samples u and v, you need to use sample_sets = [[u], [v]]?
Would this be different to the interpretation of K_c0 in Speed & Balding?

Reply:
Right, something like this. 

And, right - K_c0 is only written for diploids.